src/tools/clippy/clippy_lints/src/methods/needless_collect.rs

```diff
@@ -127,7 +127,7 @@ pub(super) fn check<'tcx>(
     }
 }
 
-/// checks for for collecting into a (generic) method or function argument
+/// checks for collecting into a (generic) method or function argument
```

Comment:
Shouldn't this change go into https://github.com/rust-lang/rust-clippy/ instead?

Reply:
yeah,I will revert this part

Reply:
fixed @clubby789